### PR TITLE
[Core] register missing elements

### PR DESCRIPTION
**Pull request by ddiezrod**
I just need this for a test, but I think it makes sense to have them registered..

### CODE REVIEW

**Comment by pooyan-dadvand** [comment]
I agree, but leaving it to @jcotela or @RiccardoRossi to approve

**Comment by jcotela** [comment]
I have no opinion either for or against adding these elements. I would just suggest registering them according to the "dimension+num_nodes" convention (so `DistanceCalculationElementSimplex2D3N` and so on)

**Comment by ddiezrod** [comment]
> I have no opinion either for or against adding these elements. I would just suggest registering them according to the "dimension+num_nodes" convention (so `DistanceCalculationElementSimplex2D3N` and so on)

agree!

**Comment by philbucher** [comment]
I thought those elements could be moved to an application at some point, or is there a reason for them to be in the core?

If they are used in the core then please go ahead

**Comment by ddiezrod** [comment]
@philbucher they are used in the redistance and level set convection processes inside the core

File: kratos/elements/distance_calculation_element_simplex.h [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
 // Application includes
 #include "includes/variables.h"
+#include "includes/kratos_flags.h"
```
Comment (philbucher) [inline kratos/elements/distance_calculation_element_simplex.h:36]: Why the additional incudes?

File: kratos/elements/levelset_convection_element_simplex.h [hunk 2]
```diff
@@ -27,6 +27,7 @@
 #include "includes/variables.h"
 #include "includes/serializer.h"
 #include "includes/cfd_variables.h"
+#include "includes/convection_diffusion_settings.h"
```
Comment (philbucher) [inline kratos/elements/levelset_convection_element_simplex.h:30]: Same

**Review by philbucher — COMMENTED** [review]
Minor comments then I approve

File: kratos/elements/distance_calculation_element_simplex.h (hunk 1 above)
Comment (ddiezrod) [reply kratos/elements/distance_calculation_element_simplex.h:36]: you get a compilation error without it as it uses BOUNDARY flag (kratos_flags is not included in kratos_components either)

File: kratos/elements/levelset_convection_element_simplex.h (hunk 2 above)
Comment (ddiezrod) [reply kratos/elements/levelset_convection_element_simplex.h:30]: similar to above one

File: kratos/elements/distance_calculation_element_simplex.h (hunk 1 above)
Comment (philbucher) [reply kratos/elements/distance_calculation_element_simplex.h:36]: Ok last question, is it needed in the header or could it be in the cpp?

Comment (ddiezrod) [reply kratos/elements/distance_calculation_element_simplex.h:36]: there is no cpp for these two elements they are both header only

**Review by philbucher — APPROVED** [review]

**Comment by ddiezrod** [comment]
Thanks! @philbucher

**Pull request merged**